admin/assets/javascripts/discourse/routes/admin-plugins-show-discourse-ai-tools-show.js

```diff
@@ -1,17 +1,18 @@
 import DiscourseRoute from "discourse/routes/discourse";
 
-export default DiscourseRoute.extend({
+export default class DiscourseAiToolsShowRoute extends DiscourseRoute {
   async model(params) {
     const allTools = this.modelFor("adminPlugins.show.discourse-ai-tools");
     const id = parseInt(params.id, 10);
-    return allTools.findBy("id", id);
-  },
 
-  setupController(controller, model) {
-    this._super(controller, model);
+    return allTools.find((tool) => tool.id === id);
```

Comment:
@martin-brennan and you need to have a chat :) he keeps pushing me to use findBy ... I don't really care, I do get that find is a fraction of a micro second faster.